www/javascript/xml-rpc-ajax.js

```diff
@@ -434,31 +430,33 @@ XML_RPC_Request.getXmlRpcValue = function(value, type)
  *
  * @return string the XML-RPC type of the JavaScript variable.
  */
-XML_RPC_Request.getType = function(value)
-{
+XML_RPC_Request.getType = function (value) {
   switch (typeof value) {
-  case 'string':
-    return 'string';
-  case 'boolean':
-    return 'boolean';
-  case 'number':
-    // this is a best guess
-    return 'double';
-  // both array and objects are 'object'
-  case 'object':
-    if (value instanceof Array)
-      return 'array';
-
-    if (value instanceof Date)
-      return 'date';
-
-    return 'struct';
-  default:
-    throw new XML_RPC_Exception(0, 'XML_RPC_Request: Cannot convert a ' +
-      'value of type ' + typeof value + ' to a valid XML-RPC ' +
-      'request type.');
+    case 'string':
+      return 'string';
+    case 'boolean':
+      return 'boolean';
+    case 'number':
+      // this is a best guess
+      return 'double';
+    // both array and objects are 'object'
+    case 'object':
+      if (value instanceof Array) return 'array';
```

Comment:
Can these get braces?

Reply:
I'm surprised prettier didn't change this ... manually fixing.

Reply:
(I guess it would be an eslint thing, but we're not using that.)